FixedSessionFaker.swift

```diff
@@ -0,0 +1,73 @@
+// Created by Lunar on 15/09/2021.
+//
+
+import Foundation
+import CoreData
+
+class FixedSessionFaker {
+    init(context: NSManagedObjectContext) {
+        let req: NSFetchRequest<SessionEntity> = SessionEntity.fetchRequest()
+        req.predicate = NSPredicate(format: "name LIKE %@", "Its complicated")
+        var session: SessionEntity!
+        if let already = try! context.fetch(req).first {
+            print("A")
+            session = already
+        } else {
+            print("B")
+            session = SessionEntity(context: context)
+            session.type = .fixed
+            session.uuid = SessionUUID()
+            session.name = "Its complicated"
```

Comment:
Don't know if we want to use Strings to keep this somewhere or not 😄

Reply:
As far as I remember we created Strings to contain texts that are displayed for users, this one's not :)

Reply:
Yes. This is only a debug-tool for programmers, it's not a subject for `Strings`